packages/next/README.md

```diff
@@ -1292,7 +1292,9 @@ module.exports = {
     maxInactiveAge: 25 * 1000,
     // number of pages that should be kept simultaneously without being disposed
     pagesBufferLength: 2,
-  }
+  },
+  // optionally configure a port for the onDemandEntries WebSocket, not needed by default
+  devWebSocketPort: 3001,
```

Comment:
I just realized we have a `onDemandEntries` key, totally forgot about it 🤦‍♂️ Can you move `devWebSocketPort` under `onDemandEntries` and call it `websocketPort` 🕵️

Reply:
Then I'll merge 💯